index.html

```diff
@@ -215,8 +216,39 @@ <h4>Skills</h4><img src="./images/chevron-right-solid.svg" alt="works">
                     </ul>
                 </section>
 
-            </div>
+                <section class="form">
```

Comment:
- [ ] Kindly stick to the Figma design as much as possible by adding the background images in your Contact section, you can see the comparison between your project and the design in the screenshots below 


**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/155172064-a1c7ce56-d3a7-4134-9157-38dc61b73cbd.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/155172070-4606abdc-a528-4a99-8fe9-ec8abfdffa63.png)

Reply:
Hi @Hamzaoutdoors, I have fixed the issues you asked me to fix ;
1.  added the background images in my Contact section
2.  added some padding to my inputs.
3.  added to all input fields required attribute.
4. i have removed the min-length attribute as requested.